CHANGELOG.md

```diff
@@ -65,6 +65,19 @@
 
 ## v1.30.0 - TBD
 
+### Bigtable:
+
+**BREAKING CHANGES**:
+* `Async*` functions in `Table` that took `CompletionQueue&` as a parameter
+  have been removed. Users that need to update their code should use the
+  identically named functions which do not require a `CompletionQueue&`. If
+  users absolutely require a custom `CompletionQueue`, one can be supplied to
+  the `DataClient` used to construct the `Table`, via
+  `ClientOptions::DisableBackgroundThreads(CompletionQueue const&)`.
```

Comment:
1. Is this even worth saying?
2. The change to prefer `Options` > `ClientOptions` will be done in this release. I am making a mental note to come back and update `CHANGELOG.md` accordingly

Reply:
In general, I think "You cannot use Foo() anymore, use Bar() instead" is more friendly/useful than "Foo() was removed."  

It is also better to assume that folks have not memorized our reference manual, so they may not be familiar with every option in the library.

With those caveats, pointing folks to the *recommended* way to do things is the "Right Thing":tm:

Reply:
The change is Okay as-is, we just need to update the CHANGELOG on the PRs where we start recommending `Options{}` instead.